docs/components/workflow/workflow.rst

```diff
@@ -0,0 +1,243 @@
+DKAN Workflow
```

Comment:
Actually I can't remember why I moved it to a sperarate folder. Can't see why it can't be a one file. Maybe when I was doing some experiments with RST and Sphinx.